docs/writing_tests.md

```diff
@@ -132,6 +132,42 @@ viewing the existing constants and defaults in their respective modules.
 
 It's documented [here](./fixture_usage.md).
 
+## Cluster context
+
+Some platforms and test cases require more clusters during the test run.
+This applies for example to Provider-Client platform where test is executed
+from a client cluster but other clusters like a Provider cluster or more
+clients are accessible during test run through:
+
+- `MultiClusterConfig` in [/ocs\_ci/framework/\_\_init\_\_.py](/ocs_ci/framework/__init__.py)
+  - This class contains methods like `switch_ctx` that can directly manipulate
+with cluster contexts.
+- Cluster context managers in [/ocs\_ci/framework/\_\_init\_\_.py](/ocs_ci/framework/__init__.py)
+  - There are implemented multiple Context managers that aim to simplify the
+work with cluster contexts. Those Cluster context managers define `\_\_enter\_\_`
+and `\_\_exit\_\_` methods so it is possible to use them with `with` statement
```

Comment:
<img width="738" alt="Screenshot 2024-09-27 at 11 43 10" src="https://github.com/user-attachments/assets/2a55ac6b-6b5c-403d-aac7-c570cb0663cd">

The escape of underscores is not rendered probably as you wish.